src/cameras/CameraH264.cpp

```diff
@@ -7,13 +7,19 @@ CameraH264::CameraH264(DriveworksApiWrapper* driveworksApiWrapper, const YAML::N
                        std::string link, ros::NodeHandle* nodehandle)
   : CameraBase(driveworksApiWrapper, config, interface, link, nodehandle)
 {
+  int profile;
+  int level;
   nh_.param<int>("bitrate", bitrate_, DEFAULT_BITRATE);
-  ROS_INFO_STREAM("H264 Bitrate: " << bitrate_);
+  nh_.param<int>("profile", profile, (int)(DEFAULT_PROFILE));
+  nh_.param<int>("level", level, (int)DEFAULT_LEVEL);
+  profile_ = static_cast<NvMediaEncodeProfile>(profile);
```

Comment:
Here, prefer creating a lookup unordered map, rather than casting, like here : https://github.com/UT-ADL/nvidia_gmsl_driver_ros/blob/main/include/framework/Checks.hpp#L20